build.cake

```diff
@@ -377,7 +392,27 @@ Task("Publish-Packages")
             });
         }
     });
-    
+
+Task("Publish-Chocolatey-Package")
+    .IsDependentOn("Create-Chocolatey-Package")
+    .WithCriteria(()=> isLocal)
+    .Does(()=> 
+    {
+        var chocolateyApiKey = EnvironmentVariable("CHOCOLATEY_API_KEY");
+        if (string.IsNullOrEmpty(chocolateyApiKey))
+        {
+            throw new InvalidOperationException("Could not resolve Chocolatey API key.");
+        }
+
+        foreach(var chocoPgk in GetFiles(chocoRoot.Path.FullPath + "/*.nupkg"))
+        {
+            ChocolateyPush(chocoPgk, new ChocolateyPushSettings {
+                ApiKey = chocolateyApiKey,
+                Source = "https://chocolatey.org/api/v2/"
```

Comment:
@phillipsj the recommended URL to push to is actually `https://push.chocolatey.org/`.  This changed recently.